Makefile

```diff
@@ -288,6 +288,22 @@ chlog-preview: $(CHLOGGEN)
 chlog-update: $(CHLOGGEN)
```

Comment:
nit: do we still need the original `chlog` commands if we have our own version that is just for `aws-cwa-dev` branch?

Reply:
That can used to update the changelog for any API changes.